src/arrays/creating_arrays/hard/q4/MainTest.java

```diff
@@ -0,0 +1,78 @@
+package arrays.creating_arrays.hard.q4;
```

Comment:
I am fairly certain this one will also have a problem with if they use an array or not, so we should talk about it.

Reply:
Though we might be able to make using an array very difficult by adding scanner input specifying the number of names to be entered.